lib/api/CDataFrameTrainBoostedTreeRunner.cc

```diff
@@ -358,7 +358,8 @@ std::size_t CDataFrameTrainBoostedTreeRunner::estimateBookkeepingMemoryUsage(
     std::size_t totalNumberRows,
     std::size_t /*partitionNumberRows*/,
     std::size_t numberColumns) const {
-    return m_BoostedTreeFactory->estimateMemoryUsage(
+    // TODO Incremental training.
```

Comment:
Do you want to reference the issue, so this TODO won't get lost?

Reply:
[f55e90f](https://github.com/elastic/ml-cpp/pull/1772/commits/f55e90f939a5e446e72435c87fd0a38007525164)